src/ADNLPProblems/AMPGO14.jl

```diff
@@ -3,7 +3,7 @@ export AMPGO14
 function AMPGO14(; n::Int = default_nvar, type::Val{T} = Val(Float64), kwargs...) where {T}
   function f(x)
     n = length(x)
-    return -exp(-x[1]) * sin(2 * (pi * x[1]))
+    return -exp(-x[1]) * sin(2 * (T(pi) * x[1]))
```

Comment:
Should have been π. It shouldn't be necessary to convert it. I would expect promote and conversion to work with irrationals.

Reply:
The pull request follows up the comment  to that issue:
https://github.com/JuliaSmoothOptimizers/OptimizationProblems.jl/issues/174#issuecomment-1125464039

pi does not modify the type T indeed. Looks like the problem might come from JuliaInterval library.

Reply:
The error comes from IntervalArithmetic library. Multiplying pi by an interval, whaterver its type, returns a Float64 interval.